go.mod

```diff
@@ -45,7 +52,20 @@ require (
 	k8s.io/api v0.24.2
 )
 
-require github.com/dschmidt/go-layerfs v0.1.0 // indirect
+require (
```

Comment:
Wondering about this `require` section.

I see that we didn't import the `internal\service` this way before.

We used to have only a single import of `github.com/aws/aws-sdk-go-v2 v0.24.0`

WDYT?